### ISSUE TITLE
Since 2 hours many entities disabled

### ISSUE THREAD

**Issue by KlausV66** [BenPru/luxtronik#208]
Hello, 
pls help. 
Many entities from my heat pump alpha innotec are not more available. 
I'm using your beta version 2023.11.1. and HA in the version 2023.11.2. 
I have restarted HA and Proxmox, no success. 

Do you have any ideas? 

BR, Klaus

**Comment by KlausV66** [comment]
thanks. How about your plan to release the beta into final version?

**Comment by BenPru** [comment]
> How about your plan to release the beta into final version?

For new users without history values the beta is stable enough. But the migration from the current stable to the beta still fails...
I have to make more tests and debug it. But I don't have enough time. So I think januar is possible. :-/

**Comment by BenPru** [comment]
No, If you make an update from the last stable 2023.03.* to a beta, the data migration is not full complete.

**Comment by KlausV66** [comment]
Sorry for the question again, I would like to update from beta to beta and keep the data if possible.
